soh/soh/OTRGlobals.cpp

```diff
@@ -1540,6 +1541,11 @@ extern "C" int CustomMessage_RetrieveIfExists(GlobalContext* globalCtx) {
             }
         } else if (textId == TEXT_SCRUB_POH || textId == TEXT_SCRUB_STICK_UPGRADE || textId == TEXT_SCRUB_NUT_UPGRADE) {
             messageEntry = Randomizer_GetScrubMessage(textId);
+        } else if (CVar_GetS32("gRandomizeRupeeNames", 0) &&
+                   (textId == TEXT_BLUE_RUPEE || textId == TEXT_RED_RUPEE || textId == TEXT_PURPLE_RUPEE ||
+                   textId == TEXT_HUGE_RUPEE)) {
+            Randomizer randomizerInstance;
+            messageEntry = randomizerInstance.Randomizer_GetRupeeMessage(textId);
```

Comment:
You should either use `OTRGlobals::Instance->gRandomizer->Randomizer_GetRupeeMessage(textId)` instead of creating a new Randomizer instance just to use that function, or make that function (and the one it calls) static and just call it like this `Randomizer::GetRupeeMessage(textId)`

Reply:
Made the functions static, much better now!